instructions/hardware/jetson_nano.md

```diff
@@ -23,51 +21,60 @@ ROS is a set of libraries and tools that help people build robot software. If yo
+
+- Connect **Pin 2** of **TELEM2** (TX/out) on Cube+ to **Pin 10** of **GPIO** (UART_RX/in, /dev/ttyTHS1) on Jetson Nano
+- Connect **Pin 3** of **TELEM2** (RX/in) on Cube+ to **Pin 8** of **GPIO** (UART_TX/out, /dev/ttyTHS1) on Jetson Nano
+- Connect **Pin 6** of **TELEM2** (Ground) on Cube+ to **Pin 9** of **GPIO** (Ground) on Jetson Nano
 
-Setting up communications between the Jetson Nano in the Pixhawk requires two steps: first, wiring the two devices together, and second, configuring the appropriate communications parameters.
+<p align="center">
+<img src="../images/cube_telem2.png" width="300">
+</p>
 
-## Wiring Between the Jetson Nano and Pixhawk 4 Mini
+The above figure shows the JST-GH cable connected to TELEM2 port of the flight controller.
 
-The Pixhawk 4 Mini's **Telem 1** port can be connected to Jetson Nano's **GPIO** pins using the JST-GH-to-Dupont connector provided in your kit. This [video](https://www.youtube.com/watch?v=nIuoCYauW3s) provides instructions on how to connect the pins. Since the jumper cable is provided, you do not need to do any soldering. In summary, you will:
+## Configure Orange Cube+
 
-- Connect **Pin 2** of **TELEM1** (TX/out) on Pixhawk 4 Mini to **Pin 10** of **GPIO** (UART_RX/in, /dev/ttyTHS1) on Jetson Nano
-- Connect **Pin 3** of **TELEM1** (RX/in) on Pixhawk 4 Mini to **Pin 8** of **GPIO** (UART_TX/out, /dev/ttyTHS1) on Jetson Nano
-- Connect **Pin 6** of **TELEM1** (Ground) on Pixhawk 4 Mini to **Pin 9** of **GPIO** (Ground) on Jetson Nano
+Before the two devices can communicate with each other, we need to set some parameters on Orange Cube+. Download and install **QGroundControl (QGC)** on your host computer by following the instructions [here](https://docs.qgroundcontrol.com/master/en/getting_started/download_and_install.html). Connect the Orange Cube+ to the computer and once it shows up in QGC, set the following parameters ([tutorial](https://docs.qgroundcontrol.com/master/en/SetupView/Parameters.html)):
 
-## Configure Pixhawk 4 Mini
+- MAV_1_CONFIG -> TELEM2
 
-Before the two devices can communicate with each other, we need to set some parameters on Pixhawk 4 Mini. Download and install **QGroundControl (QGC)** on your host computer by following the instructions [here](https://docs.qgroundcontrol.com/master/en/getting_started/download_and_install.html). Connect the Pixhawk 4 Mini to the computer and once it shows up in QGC, set the following parameters ([tutorial](https://docs.qgroundcontrol.com/master/en/SetupView/Parameters.html)):
+Reboot the flight controller. Next, verify that the following parameter is set accordingly:
 
-- *SER_TEL1_BAUD* to 921600
-- *MAV_0_RATE* to 921600
+- SER_TEL2_BAUD to 921600
```

Comment:
Do they not need to set the MAV_0_RATE anymore?

Reply:
Yeah, since we are using TELEM2 setting MAV_1_RATE to 0 selects the appropriate rate for Mavlink output. We could force MAV_1_RATE to a specific value should we see any problems in the future.